Makefile

```diff
@@ -57,6 +57,15 @@ include Makefile.common
 all: all-$(guess_platform)
 check: check-$(guess_platform)
 
+# [[ MDW-2017-05-09 ]] feature_clean_target
+clean:
```

Comment:
I suggest making this clean-linux for now - as it is specific to the linux builds.